### PR TITLE
Wrote stuff about the name property

### LINKED ISSUE #793: Configuration Docs: "name" argument is not documented
By default, the app name is the package.json "name" value. But it can be overwritten in `"hoodie": { "name": "my-custom-name"}`

### PR DESCRIPTION
**Pull request by maraisr**
Resolves #793

### CODE REVIEW

File: docs/guides/configuration.rst [hunk 1]
```diff
@@ -33,6 +33,8 @@ Here is a list of all available options
 +-----------------+--------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
 | adminPassword   | -                        | ``--adminPassword``      | ``hoodie_adminPassword`` | Password to login to Admin Dashboard. Login is not possible unless set                                              |
 +-----------------+--------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
+| name            | hoodie                   | ``--name``               | ``hoodie_name``          | Name your application.                                                                                              |
++-----------------+--------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
```
Comment (gr2m) [inline docs/guides/configuration.rst]: the default value is not `hoodie`, but the value of the `"name"` property in the `package.json` file

File: docs/guides/configuration.rst [hunk 2]
```diff
@@ -54,7 +56,8 @@ default values
         "inMemory": false,
         "loglevel": "warn",
         "url": "",
-        "adminPassword": ""
+        "adminPassword": "",
+        "name": "hoodie"
```
Comment (gr2m) [inline docs/guides/configuration.rst]: maybe use `"my-hoodie-app"` as an example value? `"hoodie"` might be confusing

**Review by gr2m — CHANGES_REQUESTED** [review]
Thank you Marais! I’ve two comments, could you please look into it?

File: docs/guides/configuration.rst (hunk 1 above)
Comment (maraisr) [reply docs/guides/configuration.rst]: :boom: thanks man - mod going up shortly!

File: docs/guides/configuration.rst (hunk 2 above)
Comment (maraisr) [reply docs/guides/configuration.rst]: Definitely agree :sunglasses:! Thanks man.

**Review by gr2m — APPROVED** [review]
Looks great, thanks for the changes 👍

**Pull request merged**